### DISCUSSION TITLE
Figure of the SJER Study Area Question

### DISCUSSION THREAD

**Discussion by agneslh** [earthlab-education/Earth-Analytics-2023-01-Intro#501] [Q&A06 Vector Data]
Environment: Jupyter Notebook in code spaces.

I was trying to create a figure of the SJER Study Area in notebook 00-site-map. I initially tried to use this code (provided in the instructions):

 ```
for rttyp, gdf in roads_gdf.groupby('RTTYP'):
        madera_plot *= df.hvplot(line_color=roads_symb[rttyp], label=rttyp)'
```

but I didn't understand what the madera_plot variable was and was so ended up using this code instead:

```
#Loop through each attribute type and plot it using the colors assigned in the dictionary
for rttyp, gdf in madera_clip_gdf.groupby('RTTYP'):

    Define the color for each group using the dictionary
    color = roads_sym[rttyp]
```
    
Everything worked the way that I did it, but I'm not sure if I created my figure the right way b/c it doesn't use hvplot, it uses matplotlib.

**Comment by eculler** [comment]
@agneslh either plotting method is fine by me! 

The madera_plot variable was the first plot layer, with the plot location points in it. I was actually able to get holoviews to work using `hv.Path(roads_gdf, color='rttyp')` instead of the `for` loop, which is simpler but didn't make it in time for the instructions.
